bazel-mode.el

```diff
@@ -21,6 +21,37 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-mode nil
+  "Major mode for Bazel BUILD files."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-mode--buildifier-cmd "buildifier"
+  "Filename of buildifier executable."
+  :type 'file
+  :group 'bazel-mode)
+
+(defun buildifier ()
```

Comment:
Please prefix all names with the package name, see the second item at https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html.

Reply:
This previously did have the package name as a prefix but your previous comment on this function was "Should this be internal? I'd expect that users want to invoke it directly."